eng/PatchConfig.props

```diff
@@ -27,6 +27,7 @@ Later on, this will be checked using this condition:
   </PropertyGroup>
   <PropertyGroup Condition=" '$(VersionPrefix)' == '2.2.2' ">
     <PackagesInPatch>
+      java:signalr;
```

Comment:
@natemcmaster This look okay?

Reply:
Yes. Double check artifacts/ for the .jar file to make sure it worked.